nekoyume/Assets/_Scripts/Game/EventListener.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Nekoyume.Game
+{
+    public class EventListener : MonoBehaviour
+    {
+        public Dictionary<string, List<System.Action>> CallbackMap { get; private set; }
```

Comment:
`private Dictionary<string, List<System.Action>> CallbackMap { get; }`
How about changing this?